GSA/digitalgov.gov
# Address sitemap.xml and meta tags for search.gov

## Summary

Fixes sitemap validation issues and meta tags for search.gov indexing. Current sitemap is invalid and this is causing re-indexing issues when new content is added.

## Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-sitemap-fields/sitemap.xml)

## Solution

### Sitemap

Set namespacing for sitemap.xml hugo generation. Run [sitemap.xml](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-sitemap-fields/sitemap.xml) on https://www.mysitemapgenerator.com/ to validate XML.

Priority needs to be a value from 0.0 to 1.0. Updated code to use a decimal as hugo escapes this in the `config.yml` setup.


### Search.gov Indexing

Adds `modified_time` as per [search.gov docs](https://search.gov/indexing/metadata.html#created-date) on content pages. This will help increase our re-indexing frequency.

New field:
```
<meta property="article:modified_time" content="2024-12-31T12:25:00-0500">
```

## How To Test

1. Run sitemap on [validator link](https://www.mysitemapgenerator.com/)
2. Check that modified_time is different than published_time and exists on content pages
